pkg/test/framework/components/echo/echoboot/echoboot.go

```diff
@@ -22,16 +22,12 @@ import (
 )
 
 // NewBuilder for Echo Instances.
-func NewBuilder(ctx resource.Context) (b echo.Builder, err error) {
-	return kube.NewBuilder(ctx), nil
+func NewBuilder(ctx resource.Context) echo.Builder {
+	return kube.NewBuilder(ctx)
 }
 
 // NewBuilder for Echo Instances.
 func NewBuilderOrFail(t test.Failer, ctx resource.Context) echo.Builder {
```

Comment:
why even have this method anymore?  should we have a comment that says "this can never fail"?

Reply:
Good point, I was planning to split this change out into another PR and forgot. I'll do that

Reply:
https://github.com/istio/istio/pull/25658